### PR TITLE
Feature/send experiment result to firstore

### PR DESCRIPTION
**Pull request by Lyokolux**
## Description

Implement it

## Checklist

- [x] I compare my branch with `develop`
- [x] All tests passed after running `yarn s-check && yarn lint`

### CODE REVIEW

File: src/components/Experience.svelte [hunk 1]
```diff
@@ -13,10 +16,20 @@
...
+  let userDoc: DocumentReference<DocumentData>
+  api.currentUserDoc((v) => { userDoc = v })
+
+  const sendResults = () => {
+    const experimentResult: Experiment = {
+      id: `${displayChunkLength}-${inputChunkLength}`,
+      events
+    }
+    api.addExperimentRequest(userDoc, experimentResult)
+  }
```
Comment (johannchopin) [inline src/components/Experience.svelte:28]: ♻️ Move all this logic to `addExperimentRequest` directly. There shouldn' t be an import from firstore in this component

File: src/components/Experience.svelte [hunk 2]
```diff
@@ -13,10 +16,20 @@
...
 
   const onFilled = (): void => {
     $swiper.slideNext()
-    // TODO: Store events in firestore
+    sendResults()
```
Comment (johannchopin) [inline src/components/Experience.svelte:32]: ♻️ This component shouldn't send to database but the parent.

File: src/store/api.ts [hunk 3]
```diff
@@ -23,15 +23,14 @@ function createApi() {
...
-  // TODO: type experiment when the type will be defined
   const addExperimentRequest = async (userDoc: DocumentReference, experiment: Experiment) => (
     updateDoc(userDoc, {
       experiments: arrayUnion(experiment)
     })
   )
 
   return {
-    currentUserDoc,
+    currentUserDoc: currentUserDoc.subscribe,
```
Comment (johannchopin) [inline src/store/api.ts:33]:
```suggestion
    userDoc: currentUserDoc.subscribe,
```

**Review by johannchopin — CHANGES_REQUESTED** [review]

File: src/components/Experience.svelte (hunk 1 above)
Comment (Lyokolux) [reply src/components/Experience.svelte:28]: ✅

File: src/components/Experience.svelte (hunk 2 above)
Comment (Lyokolux) [reply src/components/Experience.svelte:32]: ✅

File: src/store/api.ts (hunk 3 above)
Comment (Lyokolux) [reply src/store/api.ts:33]: simply subscribe

**Review by johannchopin — APPROVED** [review]
Please **rebase** and then merge

File: src/components/AlphanumericInput/AlphanumericInput.svelte [hunk 4]
```diff
@@ -2,7 +2,7 @@
   import { _ } from 'svelte-i18n'
   import type { Event } from './utils'
 
-  import { screen } from '../../stores'
+  import { screen } from '../../store'
```
Comment (johannchopin) [inline src/components/AlphanumericInput/AlphanumericInput.svelte:5]: Please use a folder called `stores`:

```suggestion
  import { screen } from '../../stores'
```

Comment (Lyokolux) [reply src/components/AlphanumericInput/AlphanumericInput.svelte:5]: As there is many stores in it, yes :+1:

Comment (Lyokolux) [reply src/components/AlphanumericInput/AlphanumericInput.svelte]: ✅

**Pull request merged**